core/controllers/editor.py

```diff
@@ -548,6 +785,19 @@ class StateInteractionStatsHandler(EditorHandler):
     """Returns detailed learner answer statistics for a state."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        },
+        'escaped_state_name': {
```

Comment:
Could we verify somehow that it is really escaped?

Reply:
Not able to add anything extra, kindly help in this.

Reply:
Hmm, could you check what the name actually means and what functions are applied to it in the frontend? From that we can discuss what verifications can be enabled.

Reply:
It is a state_name in the frontend but in the backend it is referenced as escaped_state_name.
[Reference to frontend](https://github.com/oppia/oppia/blob/82e41b10482740b12c60a35fd57a7521bfc113bb/core/templates/domain/exploration/state-interaction-stats-backend-api.service.ts#L38)  
[Reference to main.py](https://github.com/oppia/oppia/blob/82e41b10482740b12c60a35fd57a7521bfc113bb/main.py#L646)

Reply:
Okay, maybe use just `state_name` in the backend too.

Reply:
Updated.